tests/push/test_email.py

```diff
@@ -305,6 +307,36 @@ def test_encrypted_message(self):
         # We should get emailed about that message
         self._check_for_mail()
 
+    def test_no_email_sent_after_removed(self):
+        # Create a simple room with two users
+        room = self.helper.create_room_as(self.user_id, tok=self.access_token)
+        self.helper.invite(
+            room=room, src=self.user_id, tok=self.access_token, targ=self.others[0].id
+        )
+        self.helper.join(room=room, user=self.others[0].id, tok=self.others[0].token)
```

Comment:
Note that when specifying argument names for all arguments, it can often be more readable to write each on a separate line. I found this rather tricky to scan through.

Unless black is forcing it to be on one line, then :shrug: